### ISSUE TITLE
If condition for `registration` parameter

### ISSUE THREAD

**Issue by luiztauffer** [mikarubi/voluseg#12]
The values for the parameter `registration` are usually strings:

https://github.com/mikarubi/voluseg/blob/2c4d935c032c6cd4bf096990d13be79721669888/voluseg/_tools/parameter_dictionary.py#L6

So this if condition will always be satisfied, the code under else will never run:

https://github.com/mikarubi/voluseg/blob/2c4d935c032c6cd4bf096990d13be79721669888/voluseg/_steps/step1.py#L92-L96

In which cases the `else` condition should run?

**Comment by mikarubi** [comment]
correct. the idea would be to pass None, which is currently done as a string for consistency.
